packages/mdc-chips/README.md

```diff
@@ -81,6 +81,8 @@ You can optionally add a leading icon (i.e. thumbnail) and/or a trailing icon to
 
 #### Trailing icon
 
+A trailing icon comes with the functionality to remove the chip from the set. If you're adding a trailing icon, also set `tabindex="0"` and `role="button"` to make it accessible by keyboard and screenreader. Trailing icons should only be added to [input chips](###input-chips).
```

Comment:
`###input-chips` should just be `#input-chips`, it's linking to another part of the page.